ecs-init/config/common_test.go

```diff
@@ -253,3 +253,30 @@ func TestAgentRunningInExternal(t *testing.T) {
 	defer os.Unsetenv(ExternalEnvVar)
 	assert.True(t, RunningInExternal())
 }
+
+func TestCredentialsFetcherUnixSocketWithoutCredentialsFetcherHost(t *testing.T) {
+	// Make sure that the env variable is not set
+	os.Unsetenv("CREDENTIALS_FETCHER_HOST")
```

Comment:
I dont think this is needed, if you've not set it, it wont be.

also using t.Setenv might help with unsurety

Reply:
making sure this variable is not set anywhere else